client/my-sites/checkout/checkout-thank-you/atomic-store-thank-you-card.jsx

```diff
@@ -116,7 +116,7 @@ class AtomicStoreThankYouCard extends Component {
 			<div className="checkout-thank-you__atomic-store-action-buttons">
 				<a
 					className={ classNames( 'button', 'thank-you-card__button' ) }
-					href={ site.URL + '/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard' }
+					href={ siteWooCommerceWizardUrl }
```

Comment:
This update doesn't fix the issue but it seemed best to align the Woo wizard link with the one used in the plugins redirect.